src/assets/js/data/emoji.json

```diff
@@ -0,0 +1,138 @@
+{
+    "frequent": [
```

Comment:
I've noticed some platforms limit the amount of emojis you can react with- do you think we should funnel down to like 5 or 6 to start with?

check out iMessage and Facebook Messenger for some examples